htdocs/admin/delais.php

```diff
@@ -289,7 +289,7 @@
 print '<br>';
 
 
-if ($conf->global->MAIN_DISABLE_METEO != 1) {
+if (isset($conf->global->MAIN_DISABLE_METEO) && $conf->global->MAIN_DISABLE_METEO != 1) {
```

Comment:
This test is not similar with previous one.
For example, if   $conf->global->MAIN_DISABLE_METEO  is not set, old test was true, new one is false